src/libs/providers.js

```diff
@@ -0,0 +1,9 @@
+export const allProviders = ['fence', 'dcf-fence', 'anvil']
```

Comment:
TOL: Ideally we could pull these out of the keys in `providerName`. Adding a note to address this in the renaming of our links. (Something like `_.keys(providerName...).alphabetize` Understanding that alphabetize may not be a function we get for free.)